.travis.yml

```diff
@@ -10,7 +10,8 @@ env:
     global:
         - TRAVIS_PAR_MAKE="-j 4"
         - SOS_GLOBAL_BUILD_OPTS="--enable-picky --enable-pmi-simple FCFLAGS=-fcray-pointer"
-        - PTL_IFACE_NAME=venet0
+        #- PTL_IFACE_NAME=venet0
```

Comment:
Do we need to keep this commented line?

Reply:
I left it there as a note, since I don't think I'll remember it if we ever decide to enable more Portals testing in Travis.

Reply:
For some reason, Portals isn't working on venet0 any more so I set it to use loopback, which is fine for single PE tests.

Reply:
ok, thanks